doc/api/errors.md

```diff
@@ -763,6 +763,11 @@ The selected public or private key encoding is incompatible with other options.
 
 An invalid [crypto digest algorithm][] was specified.
 
+<a id="ERR_CRYPTO_INVALID_KEY_OBJECT_TYPE"></a>
+### ERR_CRYPTO_INVALID_KEY_OBJECT_TYPE
+
+The given crypto key object has an invalid type.
```

Comment:
Worth a bit more text, I think. The key object itself doesn't have an invalid type (those can't be created), its that the type is not valid for the method its being used for, I assume. A sym key for a crypto.Sign(), for example.